rn/Teacher/ios/Teacher/Submissions/QuizSubmissionList/View/QuizSubmissionListItemView.swift

```diff
@@ -0,0 +1,82 @@
+//
+// This file is part of Canvas.
+// Copyright (C) 2023-present  Instructure, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+//
+
+import Core
+import SwiftUI
+
+public struct QuizSubmissionListItemView: View {
+    private var model: QuizSubmissionListItemViewModel
+    private var action: () -> Void
+
+    public init(model: QuizSubmissionListItemViewModel, action: @escaping () -> Void) {
+        self.model = model
+        self.action = action
+    }
+
+    public var body: some View {
```

Comment:
Minor one but the selection indicator seems to be different compared to what we have on other screens. We could use `ContextButton` as `buttonStyle` but it might be the `List` that controls this.

Reply:
I tried a few of these, couldn't figure it out. since it's not really selectable, I'll let this one go